scaling_performance/host_practices.adoc

```diff
@@ -136,6 +136,62 @@ endif::[]
 etcd processes are typically memory intensive. Master / API server processes are CPU intensive. This makes them a reasonable co-location pair within a single machine or virtual machine (VM). Optimize communication between etcd and master hosts either by co-locating them on the same host, or providing a dedicated network.
 ====
 
+[[providing-storage-to-an-etcd-node-using-pci-passthrough]]
+=== Providing Storage to an etcd Node Using PCI Passthrough
+
+To provide fast storage to an etcd node so that etcd is stable at large scale,
+use PCI passthrough. To set this up with Red Hat OpenStack 11 or later:
+
+. Ensure Intel Vt-x is enabled in BIOS.
+
+. Enable the input–output memory management unit (IOMMU). In the
+*_/etc/sysconfig/grub_* file, add `intel_iommu=on iommu=pt` to the end of the
+`GRUB_CMDLINX_LINUX` line, within the quotation marks.
+
+. Regenerate *_/etc/grub2.cfg_* by running:
++
+----
+$ grub2-mkconfig -o /etc/grub2.cfg
+----
+
+. Reboot the system.
```

Comment:
Above steps need to be done only on the OpenStack Compute Nodes where the PCI device exists